xray/commands/curation/audit.go

```diff
@@ -74,25 +74,23 @@ type PackageStatus struct {
 }
 
 type Policy struct {
```

Comment:
Can we add "omitempty" to all JSON fields in the structs in this file?

Reply:
added to some.